packages/core/src/services/datasets/create.ts

```diff
@@ -22,15 +24,47 @@ export const createDataset = async (
     }
     disk: DiskWrapper
```

Comment:
this is called disk but it can be other file storage no? like s3 or others

Reply:
This is how they called in flydrive 
![image](https://github.com/user-attachments/assets/c67c54b9-fa18-47f2-a40d-c5070f75fe38)